language/Italian/strings.po

```diff
@@ -2475,11 +2475,11 @@ msgstr "Impostazioni"
 
 #: plugins/timer/init.lua:94
 msgid "Current time"
-msgstr ""
+msgstr "Ora attuale"
```

Comment:
This is used for the the play time for the current session - is this translation appropriate?  (The English is not particularly clear for this string.)